### PR TITLE
Add lobby

### PR DESCRIPTION
**Pull request by Chroq**
## Description

Trello: https://trello.com/c/3W8vHuQw/4-2-as-norbert-i-want-to-play-online-with-philippe

Get back all available games in dedicated modals.

No game available:
image

Available games:
image

### CODE REVIEW

File: components/menus/MainMenu.js [hunk 1]
```diff
@@ -25,6 +26,13 @@ export default function MainMenu(props) {
         justify="center"
         m="2%"
       />
+      <OnlineRejoinModal
+        width="100%"
+        alignItems="center"
+        justify="center"
```
Comment (djhi) [inline components/menus/MainMenu.js:32]: Those two properties targets the OnlineRejoinModal children

File: pages/api/games.js [hunk 2]
```diff
@@ -12,5 +12,15 @@ export default (req, res) => {
       .then((game) => {
         res.status(200).json(game);
       });
+  } else if (req.method === "GET") {
```
Comment (djhi) [inline pages/api/games.js:15]: It more a matter of preferences but you could rewrite this function using switch and async/await. Besides, you could separate the post and get in two separate functions, making the default merely a poor man router ;)

File: components/menus/MainMenu.js (hunk 1 above)
Comment (Chroq) [reply components/menus/MainMenu.js:32]: Fixed, TY

File: pages/api/games.js (hunk 2 above)
Comment (Chroq) [reply pages/api/games.js]: Ok, i did the half of the job, if I understand well your advice. I didn't touch to async/await but I did split the function in 3.

File: pages/api/games.js [hunk 3]
```diff
@@ -1,26 +1,38 @@
...
 };
+
+function get(res) {
+  getGameRepository()
+    .findAll({
+      where: {
+        player2_nickname: null,
+      },
+    })
+    .then((game) => {
```
Comment (djhi) [inline pages/api/games.js:22]: You forgot to handle the error cases

Comment (Chroq) [reply pages/api/games.js:22]: Add catch function to manage error from the request. Clearly missing a more robust error handling system...

**Pull request merged**